### DISCUSSION TITLE
GraphQL query returns ERROR for API Key with group for only Public Content

### DISCUSSION THREAD

**Discussion by aapwikip** [requarks/wiki#6907] [Help / Questions]
### API Key with limited Access for Public Content

- Can NOT create an API Key for Guests Group
- Create another User Group 'PublicContent' similar to Guests i.e. Content is visible to all Public
      - ReadPages & ReadAssets permissions for public pages 
- Created API Key with this 'PublicContent' group 
- But Gives Error
Error: INTERNAL_SERVER_ERROR
Message: PageViewForbidden 6013: You are not authorized to view this page

{"errors":[{"message":"You are not authorized to view this page.","locations":[{"line":3,"column":5}],"path":["pages","single"],"extensions":{"code":"INTERNAL_SERVER_ERROR","exception":{"stack":"PageViewForbidden 6013: You are not authorized to view this page.\n    at single (/home/xxx/xxx/server/graph/resolvers/page.js:167:17)\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)","code":6013,"message":"You are not authorized to view this page."}}}],"data":{"pages":{"single":null}}}

### API Key with FULL access

Works Great

**Comment by NellsRelo** [comment]
Mentioned in #6216 , mentioning here as well - you'll need the Manage Page permissions for your API key - that seems to be the permission graphql is looking for
